spec/unit/local_system_spec.rb

```diff
@@ -94,41 +94,49 @@
   end
 
   describe ".validate_machinery_compatibility" do
-    it "returns true on hosts that can run machinery" do
-      allow(LocalSystem).to receive(:os).and_return(OsOpenSuse13_1.new)
+    context "on hosts that can run machinery" do
+      it "returns true" do
+        allow(LocalSystem).to receive(:os).and_return(OsOpenSuse13_1.new)
 
-      expect {
-        LocalSystem.validate_machinery_compatibility
-      }.not_to raise_error
+        expect {
+          LocalSystem.validate_machinery_compatibility
+        }.not_to raise_error
```

Comment:
The method never raises an exception, so that test does not make sense. You should rather check that `captured_machinery_output` is empty.